include/mppic/trajectory_visualizer.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_lifecycle/lifecycle_node.hpp"
+#include "visualization_msgs/msg/marker_array.hpp"
+#include "tf2_geometry_msgs/tf2_geometry_msgs.hpp"
+#include "nav_msgs/msg/path.hpp"
+
+namespace mppi::visualization {
+class TrajectoryVisualizer
+{
+public:
+  TrajectoryVisualizer() = default;
+
+  void on_configure(rclcpp_lifecycle::LifecycleNode::WeakPtr parent, const std::string & frame_id);
+  void on_cleanup();
+  void on_activate();
+  void on_deactivate();
+
+  void reset();
+
+  template <typename Container> void add(Container && trajectory);
```

Comment:
Definition must be here, otherwise it wouldn't work. Other approach is to create impl hpp file

Reply:
Merged. i'll try to fix it if it breaks.